src/app/pages/TopicPage/Curation/index.jsx

```diff
@@ -34,6 +35,7 @@ const Curation = ({
   position,
   curationLength,
 }) => {
+  if (visualStyle === VISUAL_STYLE.BANNER) return null;
```

Comment:
I think we should either move this logic out of here and into TopicPage, OR since curation is really only built to handle simple / hierarchical grid, which both have a visual style of `NONE` (or the default value), then we should return null if visual style is not `NONE` ?

```suggestion
  if (visualStyle !== VISUAL_STYLE.NONE) return null;
```